bitnami/kubeapps/templates/dashboard/configmap.yaml

```diff
@@ -2,17 +2,24 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ template "kubeapps.dashboard-config.fullname" . }}
-  labels:{{ include "kubeapps.extraAppLabels" . | nindent 4 }}
-    app: {{ template "kubeapps.dashboard-config.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: dashboard
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" . ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
 data:
   vhost.conf: |-
     server {
-      listen {{ .Values.dashboard.service.port }};
+      listen {{ .Values.dashboard.containerPort }};
```

Comment:
I guess this change isn't a breaking change? (Unless I've set service.port to a non-default value in my values.yaml, but then it should error anyway if that's no longer an option).

Reply:
Correct!
The idea is basically difference between the port you used in the container, and the port you expose in the service. In most cases both are set to the same value, but this opens the possibility to customize then independently. I know there are people that use random ports at container level for security reasons.